grammars/javascript.cson

```diff
@@ -724,6 +724,41 @@
     ]
   }
   {
+    # Math
+    'begin': '\\bMath\\b'
+    'beginCaptures':
+      '0':
+        'name': 'entity.name.type.object.math.js'
+    'end': '(?!\\G)'
+    'patterns': [
+      {
+        # Math.random()
+        'begin': '\\s*(\\.)\\s*(abs|acos|acosh|asin|asinh|atan|atan2|atanh|cbrt|ceil|clz32|cos|cosh|exp|expm1|floor|fround|hypot|imul|log|log10|log1p|log2|max|min|pow|random|round|sign|sin|sinh|sqrt|tan|tanh|trunc)\\s*(?=\\()'
+        'beginCaptures':
+          '1':
+            'name': 'meta.delimiter.method.period.js'
+          '2':
+            'name': 'support.function.math.js'
+        'end': '(?<=\\))'
+        'name': 'meta.method-call.js'
+        'patterns': [
+          {
+            'include': '#arguments'
+          }
+        ]
+      }
+      {
+        # Math.PI
+        'match': '\\s*(\\.)\\s*(E|LN10|LN2|LOG10E|LOG2E|PI|SQRT1_2|SQRT2)\\b'
```

Comment:
What scope should be used instead of `support.property.math.js`? The old scope was `support.constant.js`.

PI is both a constant _and_ a property of the Math object, so `support.property.constant.math.js`?